PROJECT_GUIDELINES.md

```diff
@@ -0,0 +1,228 @@
+# Project Guidelines
+
+Hey there and thanks for stopping by! We welcome all feedback and contributions to [Calcite App Components](https://github.com/Esri/calcite-app-components/blob/master/README.md). If you are interested in getting involved, we ask that you give our guidelines a quick read through. Below is a list of things we've found work well for us as we continue developing our components.
+
+## TOC
+
+- [Code of conduct](https://github.com/Esri/contributing/blob/master/CODE_OF_CONDUCT.md)
+- [Contribution guidelines](https://github.com/esri/contributing)
+- [Conventions](#conventions)
+  - [Formatting](#formatting)
+  - [Github](#github)
+  - [Report a bug!](#report-a-bug)
+- [Dist/package](https://github.com/Esri/calcite-app-components/blob/master/GETTING_STARTED.md_)
+- [Code base](#code-base)
+  - [A11y](#a11y)
+  - [Components](#components)
+- [Gotchas](#gotchas)
+
+## Conventions
+
+### Formatting
+
+This project uses lint-staged to automatically format code on commit, making it easier to contribute.
```

Comment:
Add link to `lint-staged` or just mention that the project is set up to auto format on commit.